mldsa/mldsa_native_config.h

```diff
@@ -655,13 +655,11 @@
  *
  * Description: Set this to reduce RAM usage.
  *
- *              This configuration option is work in progress.
- *
- *              At present it results the following memory saving in signing
- *              with no impact on performance:
- *               - ML-DSA-44: 4 KiB
- *               - ML-DSA-65: 5 KiB
- *               - ML-DSA-87: 7 KiB
+ *              This trades memory for performance:
+ *              - Memory savings: 16 KB (ML-DSA-44), 30 KB (ML-DSA-65),
+ *                56 KB (ML-DSA-87)
```

Comment:
We should split this by keygen, signing, verificaton. 
If I understand correctly, the current saving is for signing, while for the others we save 12/25/49.